howto/manage/logs.md

```diff
@@ -99,3 +99,22 @@ $  juju model-config logging-config="<root>=WARNING;unit=TRACE"
 It will set the logging level for all units to `TRACE`.
 
 [note type="caution" status="Warning"]It isn't a good idea to leave the logging level at `TRACE` for any longer than you actually need to. Verbose logging not only consumes network bandwidth but also fills up the database on the controller.[/note]
+
+## Aggregating logs
+
+When monitoring your deployment, it is a good idea to centralise all logs in a log aggregator to spot issues early.
+Below is a non-exhaustive list of useful log files as well as which machine they can be found on.
+
+| Log file location/command                                                                                                             | Machine                                                                                                                  | Description                                                         |
+|---------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|
+| `/var/snap/lxd/common/lxd/logs/ams-*/console.log`                                                                                     | Any LXD node                                                                                                             | LXD logs for a specific container                                   |
+| `journalctl -u anbox.service`                                                                                                         | Within an AMS container (accessible from an addon or an agent installed in the container), or by running `amc logs <id>` | Anbox logs                                                          |
+| `/var/lib/anbox/logs`                                                                                                                 | Within an AMS container (accessible from an addon or an agent installed in the container), or by running `amc logs <id>` | Container-related logs for a single container                       |
```

Comment:
For container-related logs, `amc logs <id>` doesn't help here as it only outputs
. Android related logs with `amc logs -t android <id>`
. Anbox related logs with `amc logs <id>`